be/src/column/field.h

```diff
@@ -157,6 +161,9 @@ class Field {
 
     ColumnPtr create_column() const;
 
+    void set_uid(ColumnUID uid) { _uid = uid; }
+    ColumnUID uid() const { return _uid; }
```

Comment:
`const ColumnUID& uid() const { return _uid; }`

Reply:
ColumnUID is `int32_t`, not need this?

Reply:
It's a style recommendation.

Reply:
updated.